src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/Footer.tsx

```diff
@@ -11,8 +12,10 @@ export const Footer: React.FC = () => {
 
   const trackStartSellingClick = () => {
     trackEvent({
-      action: "clickedStartSelling",
-      context_module: "Footer",
+      action: ActionType.tappedConsign,
```

Comment:
@audrey-thenot "clickedStartSelling" is new, so it is safe to change it to the one from cohesion

Reply:
ok

Reply:
"Footer"